promlog/log.go

```diff
@@ -18,12 +18,23 @@ package promlog
 
 import (
 	"os"
+	"time"
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 )
 
+var (
+	timestampFormat = "2006-01-02T15:04:05.000Z07:00"
+)
+
+func init() {
```

Comment:
You're still affecting all users of the library here. Can you make it so it only affects our loggers?

Reply:
I don't think so, the global var is the only way to specify the timestamp format in the go-kit/log lib.

Reply:
You can change it 3 lines from the bottom of this file

Reply:
Indeed